WordPress/openverse-frontend
# Fix homepage zooming on iPhone

## Fixes




Fixes #2151 by @panchovm 

## Description



This PR sets the input text to use `paragraph-large` style (16px, line-height 190%) for widths less than `md`.

`paragraph-large` style was using `leading-larger`, which was causing not-found errors during build. I replaced it with `leading-[190%]`. The line-height tailwind styles need to be fixed and will be fixed in later PRs.

Also, WebStorm complains that `semibold` is incorrect, so I replaced it with `semi-bold` :)

## Testing Instructions

Use iPhone and search for something from the homepage. You should not see the zooming as in the description of the linked issue.
![iPhoneZooming](https://user-images.githubusercontent.com/15233243/217733750-ad7c3ed0-a8cb-4e08-8da5-b6aa14d00dd1.gif)


## Checklist



      `Update index.md`).
      a parent feature branch.
      errors.

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin



<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
